tests/smt/smt_server_install.pm

```diff
@@ -147,24 +143,26 @@ sub run {
     wait_screen_change { send_key "alt-f" };
     wait_serial("$module_name-0", 200) || die "yast2 ca_mgm failed";
 
+
     #mirroring repos
+    select_serial_terminal;
+    systemctl('restart apache');
     assert_script_run "df -h";    #mirroring needs quite a lot of space
```

Comment:
I suggest using `record_info('List disk space', script_output('df -h'));`

Reply:
I would do it if it wouldn't run via serial terminal, it's nearly the same without the info square which does not work very well with long text anyway. :innocent: 
https://dzedro.suse.cz/tests/1051#step/smt_server_install/41